app/javascript/components/pages/HomePage.js

```diff
@@ -1,9 +1,30 @@
 import React, { Component } from 'react'
+import {
+  Card,
+  CardImg,
+  CardImgOverlay,
+  CardTitle,
+  CardText,
+} from "reactstrap";
 
 class HomePage extends Component {
   render() {
     return (
-      <div>Footer</div>
+      
+<div>
+  <Card inverse>
+    <CardImg
+      alt="Card image cap"
+      src="https://pbs.twimg.com/media/FaioV3DVEAAuy0l?format=jpg&name=large"
+      style={{
+        height: 900
+
+      }}
+      width="100%"
+    />
+  </Card>
+</div>
```

Comment:
This whole section should be indented over several tabs since it is nested inside the return